src/scripts.js

```diff
@@ -55,8 +59,44 @@ const filterByTag = (tag) => {
   return allRecipes.filterByTag(tag)
 }
 
-goButton.addEventListener('click', getSearchTerm);
-// *** END Nikki's work ***
+const displayLanding = () => {
```

Comment:
Looks like some great functionality here! I did move a bunch of stuff around in `scripts.js` last night but this seems isolated so hopefully the merge won't be too bad.

Reply:
we can work through it together.